### ISSUE TITLE
Python scripts with no methods

### ISSUE THREAD

**Issue by itsmarcy** [hgsc-project-managers/ngsi-pm#22]
When packaging ngsi-pm, I noticed that some scripts do not have a `main` or any methods and therefore cannot be called through a cli like the others.

Specifically:
- `dump_rgs.py`
- `dump_xl_bam_paths.py`
- `dump_xl_barcodes.py`
- `dump_xl_cram_paths.py`

Should these be rewritten as proper python scripts with a main method (therefore spawning another issue that must be done before #10 ) or should they be moved to a `bin` or `scripts` directory under the python package portion and leave them as intended?

**Comment by walkerh** [comment]
Non-conformant scripts should be refactored to support entry-points — have `main` functions.

- Supports `setup.py`
- More testable
- Solves PATH issues

**Comment by itsmarcy** [comment]
Closing as this is now solved by PR #23
